tdps.lic

```diff
@@ -37,9 +37,6 @@ stats = [DRStats.strength, DRStats.reflex, DRStats.agility, DRStats.charisma, DR
 race = DRStats.race.downcase.sub(' ', '').sub("'", '')
 mods = races.fetch(race)
 
-put 'exp all'
-waitfor('EXP HELP for more information')
-
 # Capture and extract data
 data = reget(39)
 data.each { |line| exp << line if (/\(.?.\/34\)/ =~ line) }
```

Comment:
I think line 40 is actually needed because line 44 parses the output, are you sure this needs to be deleted?